static/src/javascripts/projects/common/modules/experiments/tests/video-series-page.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+
+        this.id = 'VideoSeries';
+        this.start = '2016-04-21';
+        this.expiry = '2016-04-25';
+        this.author = 'James Gorrie';
+        this.description = 'New video series page (initial numbers test)';
+        this.audience = 1.0;
+        this.audienceOffset = 0.0;
+        this.successMeasure = '';
+        this.showForSensitive = true;
+        this.audienceCriteria = '';
+        this.dataLinkNames = '';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            return config.page.contentType === 'Video' && config.page.seriesTags;
```

Comment:
What's the behavior when we have the first item in a series and no other videos to show? Is it hard to exclude these here?